README.md

```diff
@@ -101,4 +105,46 @@ For Mod 4 this is in /home/ec2-user/to-do_app_setup on Ansible controller IP: 18
 
 view in browser at port 5000 of managed server for example:
 
-`http://3.9.36.242:5000/`
\ No newline at end of file
+`http://3.9.36.242:5000/`<br>
+<br>
+
+
+# Module 5 - Working With Docker
+
+## To build Docker Container
+```bash
+# single version
+docker build --tag todo-app .
```

Comment:
Now that you have a multistage Dockerfile, it doesn't make much sense to keep instructions for the "single version". It will build the final stage in the Dockerfile (development for you at the moment), but that's not something you want the reader to rely on.